scrunch/datasets.py

```diff
@@ -544,6 +544,8 @@ def move_here(self, items, **kwargs):
             item.url: {} for item in items
         }, graph=graph))
         self.resource.refresh()
+        for item in items:
```

Comment:
This is the scrunch bug fix. After moving the items they should reflect their new location. Without this you'd move a dataset and then do `.project` and it'd be pointing to the old location.